pkg/operatorstatus/operatorstatus.go

```diff
@@ -78,26 +80,27 @@ func (o *ClusterOperatorHandler) UpdateOperatorStatus(cfg *v1.Config) error {
 		return err
 	}
 
-	progressing, msgForProgressing := cfg.ClusterOperatorStatusProgressingCondition(stateForPending, available)
+	progressing, msgForProgressing := cfg.ClusterOperatorStatusProgressingCondition(msgForProgressing, available)
 	err = o.setOperatorStatus(configv1.OperatorProgressing,
 		progressing,
 		msgForProgressing,
-		cfg.Status.Version)
+		"")
```

Comment:
similarly we aren't saying we are progressing to given version any more?

Reply:
same answer..the reason for being in progressing should still contain the "trying to achieve version $currentVersion" message.

Reply:
same response as above

Reply:
Nope I was wrong ... needed to update my deployment